test/blackbox-tests/test-cases/all-alias.t/run.t

```diff
@@ -36,3 +36,39 @@
   $ dune build --display short --root install-alias @all
   Entering directory 'install-alias'
           echo foo
+
+@all does not depend directly on file copies from the source tree
+
+  $ mkdir -p source-file-copies
+  $ cd source-file-copies
+  $ cat > dune-project <<EOF
+  > (lang dune 3.0)
+  > EOF
+
+Add two files
+
+  $ touch a.ml b.ml
+
+An empty project, should not copy any file.
+
+  $ dune build --root .
```

Comment:
The `--root .` is superfluous here.

Reply:
Thanks, I removed it, and will merge once CI is green.